apps/backend/src/authn/okta.strategy.ts

```diff
@@ -26,50 +25,54 @@ export class OktaStrategy extends PassportStrategy(Strategy, 'okta') {
   ) {
     super(
       {
-        issuer: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default`,
-        authorizationURL: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default/v1/authorize`,
-        tokenURL: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default/v1/token`,
-        userInfoURL: `https://${
-          configService.get('OKTA_DOMAIN') || 'disabled'
-        }/oauth2/default/v1/userinfo`,
+        issuer:
+          configService.get('OKTA_ISSUER_URL') ||
+          `https://${configService.get('OKTA_DOMAIN')}` ||
+          'disabled',
+        authorizationURL:
+          configService.get('OKTA_AUTHORIZATION_URL') ||
+          `https://${configService.get('OKTA_DOMAIN') || 'disabled'}/oauth2/v1/authorize`,
+        tokenURL:
+          configService.get('OKTA_TOKEN_URL') ||
+          `https://${configService.get('OKTA_DOMAIN') || 'disabled'}/oauth2/v1/token`,
+        userInfoURL:
+          configService.get('OKTA_USER_INFO_URL') ||
+          `https://${configService.get('OKTA_DOMAIN') || 'disabled'}/oauth2/v1/userinfo`,
```

Comment:
If disabled is used, the webpage won't load, right?

Reply:
Nominally the user has been told in the docs that OKTA_DOMAIN is a required envvar if they're using this login method so we technically don't need to do anything tho I agree it's not the best experience.  We should probably have some sorta configuration vetting process that happens on spinup to ensure that all vars are set properly, but I think that'd be outside of the scope of this pr.